src/index.js

```diff
@@ -97,3 +84,57 @@ document.addEventListener('click', (e) => {
     }
   }
 });
+
+function createDeleteBtn() {
+  const createDeleteBtn = document.createElement('input');
+  createDeleteBtn.value = 'Delete';
+  createDeleteBtn.type = 'button';
+  createDeleteBtn.name = 'delBtn';
+  createDeleteBtn.classList.add('dynamicDeleteBtn');
+  return createDeleteBtn;
+}
+
+document.addEventListener('dblclick', (e) => {
+  // only when click on icon
+  if (e.target.classList.value === 'fas fa-ellipsis-v') {
+    const previous = e.target.previousSibling;
+    const parent = e.target.parentElement;
+    const { firstChild } = parent;
+
+    // add redness
+    if (parent.classList.value === 'item d-flex red') {
+      parent.classList.remove('red');
+    } else {
+      parent.classList.add('red');
+    }
+
+    if (parent.classList.value === 'item d-flex red') {
+      firstChild.classList.add('d-none');
+    } else {
+      firstChild.classList.remove('d-none');
+    }
+
+    if (parent.classList.value === 'item d-flex red') {
+      parent.insertBefore(createDeleteBtn(), e.target);
+    } else {
+      // eslint-disable-next-line no-lonely-if
+      if (previous.classList.value === 'dynamicDeleteBtn') {
+        previous.remove();
+      }
+    }
```

Comment:
- [ ]  Please use `localstorage` to store the data so that it persist after refresh as well.